main.go

```diff
@@ -0,0 +1,88 @@
+package main
```

Comment:
code is clean. as this project gets more complicated though, we need to talk about package hierarchy. check out https://github.com/spf13/cobra from the same guys who made viper.